lib/controller.js

```diff
@@ -31,6 +31,17 @@ module.exports = function createController(config) {
       query.lng = lng;
     }
 
+    if (serviceProvider.version === 'v2') {
+      query.auth_methods = serviceProvider.authMethods;
+      query.minimal_assurance_level= serviceProvider.minimalAssuranceLevel;
+    }
+
+    Object.keys(query).map(q => {
+      if (query[q] === undefined) {
```

Comment:
je mag dit van mij ook verwijderen als het null of '' is
is het dan niet voldoende als je !query[q] doet?

Een map dient hier trouwens niet voor, beter een forEach.

Ik zou q ook hernoemen naar key.